solr/core/src/java/org/apache/solr/handler/admin/MetricsHandler.java

```diff
@@ -51,24 +51,15 @@
 public class MetricsHandler extends RequestHandlerBase implements PermissionNameProvider {
   final SolrMetricManager metricManager;
 
-  public static final String COMPACT_PARAM = "compact";
-  public static final String PREFIX_PARAM = "prefix";
-  public static final String REGEX_PARAM = "regex";
-  public static final String PROPERTY_PARAM = "property";
-  public static final String REGISTRY_PARAM = "registry";
-  public static final String GROUP_PARAM = "group";
```

Comment:
This was a nice metrics filtering parameter; the most course filter and was simple to understand.  Did we need to throw this out?  I could see continuing to support this relatively easily.  On the other hand, it might strictly speaking be redundant.  And anyone who consumed metrics certainly needs to change what they do _anyway_.